packages/generator-one-app-module/generators/app/index.js

```diff
@@ -88,10 +133,110 @@ module.exports = class extends Generator {
       { globOptions: { dot: true } }
     );
 
+    if (this.moduleType === 'root module') {
+      this.fs.copyTpl(
+        this.templatePath('./root-module'),
+        this.destinationPath(),
+        {
+          modulePackageName: this.modulePackageName,
+          moduleNamePascal: this.moduleNamePascal,
+        },
+        null,
+        { globOptions: { dot: true } }
+      );
+      this.fs.extendJSON(this.destinationPath('package.json'), {
+        dependencies: {
+          'content-security-policy-builder': '^2.1.0',
+        },
+      });
+    }
+
+    if (this.setupInternationalization) {
+      this.fs.copyTpl(
+        this.templatePath('./intl-child-module'),
+        this.destinationPath(),
+        {
+          modulePackageName: this.modulePackageName,
+          moduleNamePascal: this.moduleNamePascal,
+        },
+        null,
+        { globOptions: { dot: true } }
+      );
+      this.fs.extendJSON(this.destinationPath('package.json'), {
+        dependencies: {
+          '@americanexpress/one-app-ducks': '^4.0.0',
+          immutable: '^3.8.2',
+          'prop-types': '^15.5.9',
+          'react-intl': '^3.6.0',
+          'react-redux': '^7.1.3',
+          redux: '^4.0.4',
+        },
+        devDependencies: {
+          glob: '^7.1.6',
+          '@babel/polyfill': '^7.8.3',
+          'jest-json-schema': '^2.1.0',
+
+        },
+        jest: {
+          setupFilesAfterEnv: ['./test-setup.js'],
+        },
+      });
+
+      if (this.moduleType === 'child module') {
+        this.fs.extendJSON(this.destinationPath('package.json'), {
+          'one-amex': {
+            bundler: {
+              externals: 'react-intl',
+            },
+          },
+        });
+      }
+      if (this.moduleType === 'root module') {
+        this.fs.copyTpl(
+          this.templatePath('./intl-root-module'),
+          this.destinationPath(),
+          {
+            modulePackageName: this.modulePackageName,
+            moduleNamePascal: this.moduleNamePascal,
+          },
+          null,
+          { globOptions: { dot: true } }
+        );
+        this.fs.extendJSON(this.destinationPath('package.json'), {
+          'one-amex': {
+            bundler: {
+              providesExternals: 'react-intl',
+            },
+          },
+        });
+      }
+    }
+
+    if (this.setupParrotMiddleware) {
+      this.fs.extendJSON(this.destinationPath('package.json'), {
+        'one-amex': {
+          runner: {
+            parrotMiddleware: './dev.middleware.js',
+          },
+        },
+        devDependencies: {
+          'parrot-middleware': '^3.1.0',
```

Comment:
In general, I found using the generator with the fixed package versions will make it harder to maintain and deliver the latest versions of a package, which might not reflect latest docs. For future ref, maybe we can use `npm show {package} version` to pull the `latest` tag and apply that. Will open an issue to track.

Reply:
https://github.com/americanexpress/one-app-cli/issues/29

Reply:
great idea, makes sense!